soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -47,19 +47,40 @@ const std::string Randomizer::rupeeMessageTableID = "RandomizerRupees";
 const std::string Randomizer::NaviRandoMessageTableID = "RandomizerNavi";
 const std::string Randomizer::IceTrapRandoMessageTableID = "RandomizerIceTrap";
 
-static const char* englishRupeeNames[81] = {
-    "Rupees",       "Bitcoin",       "Bananas",      "Cornflakes", "Gummybears",   "Floopies",    "Dollars",
-    "Lemmings",     "Emeralds",      "Bucks",        "Rubles",     "Diamonds",     "Moons",       "Stars",
-    "Mana",         "Doll Hairs",    "Dogecoin",     "Mushrooms",  "Experience",   "Friends",     "Coins",
-    "Rings",        "Gil",           "Pokédollars",  "Bells",      "Orbs",         "Bottle Caps", "Simoleons",
-    "Pokémon",      "Toys",          "Smackaroos",   "Zorkmids",   "Zenny",        "Bones",       "Souls",
-    "Studs",        "Munny",         "Rubies",       "Gald",       "Gold",         "Shillings",   "Pounds",
-    "Glimmer",      "Potch",         "Robux",        "V-Bucks",    "Bratwürste",   "Mesetas",     "Coal",
-    "Euro",         "Spoons",        "Cucumbers",    "FPS",        "Shekels",      "Yen",         "Canadian Dollars",
-    "Dollarydoos",  "Copper",        "Silver",       "Platinum",   "Gems",         "Minerals",    "Vespene Gas",
-    "Lumber",       "Jiggies",       "Mumbo Tokens", "KF7 Ammo",   "Remote Mines", "Credits",     "Doubloons",
-    "Ether",        "Doge",          "Cards",        "Talent",     "Poko",         "Lira",        "Kroner",
-    "Store Credit", "Social Credit", "Cocoa Beans",  "Strawbs"
+static const char* englishRupeeNames[165] = { 
+    "Bad RNG Rolls",        "Bananas",              "Beanbean Coins",       "Beans",                "Bells",
+    "Beli",                 "Berries",              "Bison Dollars",        "Bitcoin",              "Blue Essence",
+    "Bolts",                "Bones",                "Boondollars",          "Bottle Caps",          "Bratwürste",
+    "Bucks",                "BugFrags",             "Bullets",              "Bǎn",                  "Cards",
```

Comment:
I think it might be better to exclude Bullets just to make sure we keep it family friendly. KF7 Ammo already treaded that line a little in my eyes.